packages/fxa-content-server/app/styles/modules/_branding.scss

```diff
@@ -100,12 +101,6 @@
         text-align: right;
       }
     }
-
-    .firefox-family-services {
```

Comment:
The firefox family services pane is no longer displayed on this page, so I removed the CSS.